module.txt

```diff
@@ -1,11 +1,13 @@
 {
     "id": "Lakes",
     "version": "1.0.0",
-    "author": "Cpt. Crispy Crunchy",
+    "author": "Cpt. Crispy Crunchy, sin3point14",
```

Comment:
We can bring this in-line with many other modules of the basic line-up and just put "The Terasology Foundation" here (don't want to rob you of your author tag, I'll leave this comment up to you 😉 ).

Reply:
sure, done